cli/test/org/partiql/cli/CliTest.kt

```diff
@@ -128,4 +129,12 @@ class CliTest {
 
         assertEquals("{a:1}\n{b:1}\n", actual)
     }
+
+    @Test
+    fun withoutInput() {
+        val subject = makeCli("SELECT * FROM input_data")
+        val actual = subject.runAndOutput()
+
+        assertEquals("\n", actual)
+    }
 }
```

Comment:
This behavior is incorrect. If the `--input` option is not provided this query should throw an error - `No such binding found for 'input_data'`. Right now `input_data` is bound to *`emptyInputStream`*. Consider removing this binding in case the cli option `--input` is not set.

Reply:
Will apply this comment in the next commit.